acceptance-tests/tests/src/test/resources/jsonrpc/engine/test-cases/11_shanghai_prepare_payload.json

```diff
@@ -14,10 +14,16 @@
         "suggestedFeeRecipient": "0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b",
         "withdrawals": [
           {
-            "index": "0",
-            "validatorIndex": "0",
+            "index": "0x0",
```

Comment:
Interesting that didn't break anything - is that expected (i.e. is "0" a valid 0 hex value) or are we off spec and missing some validation here?

Reply:
Don't think it matters if we accept "0" as well as "0x0". The main thing is that we accept the "0x0" formatting that is normally sent. HIve tests send us hex values with 0x prefix.